src/transit.js

```diff
@@ -978,7 +978,7 @@ class Transit {
 					this.logger.debug(`=> Send stream chunk to ${nodeID} node. Seq: ${copy.seq}`);
 
 					try {
-						await this.publish(new Packet(P.PACKET_RESPONSE, nodeID, copy));
+						this.publish(new Packet(P.PACKET_RESPONSE, nodeID, copy));
```

Comment:
In this case, the `try...catch` has no affect. Please rewrite it to `this.publish(...).catch(err => { ... });` form